### ISSUE TITLE
setMode7Rot: Unexpected behavior when angle > 200

### ISSUE THREAD

**Issue by malayli** [alekmaul/pvsneslib#206]
## Description
When angle > 200, the rotation becomes unexpected.

You can test this issue by testing this example here on develop branch:
https://github.com/alekmaul/pvsneslib/tree/develop/snes-examples/graphics/Backgrounds/Mode7

**Comment by alekmaul** [comment]
ok, I think the problem is the use of byte instead of words for matrix registers
i will test it soon (i'm added a new function for mode7 scaling too)

**Comment by malayli** [comment]
@alekmaul Ok nice!

**Comment by malayli** [comment]
@jeffythedragonslayer Difficult to explain it with words. I can share a video capture on PVSnesLib Discord.

**Comment by malayli** [comment]
@jeffythedragonslayer No. Because when angle > 200, the rotation value is completely different for each frame. I let you imagine how it looks like when you run it in a rom.

**Comment by malayli** [comment]
I close this issue as I have my own values for a complete rotation.

I also think that this issue was fixed in a recent version.
